pkg/scheduler/factory.go

```diff
@@ -426,12 +425,13 @@ func (i *podInformer) Lister() corelisters.PodLister {
 
 // NewPodInformer creates a shared index informer that returns only non-terminal pods.
 func NewPodInformer(client clientset.Interface, resyncPeriod time.Duration) coreinformers.PodInformer {
-	selector := fields.ParseSelectorOrDie(
-		"status.phase!=" + string(v1.PodSucceeded) +
-			",status.phase!=" + string(v1.PodFailed))
-	lw := cache.NewListWatchFromClient(client.CoreV1().RESTClient(), string(v1.ResourcePods), metav1.NamespaceAll, selector)
+	selector := "status.phase!=" + string(v1.PodSucceeded) +
```

Comment:
nit: use `fmt.Sprintf()` to construct "selector".

Reply:
@Huang-Wei Fix done. Thx.